package/generateScss.mjs

```diff
@@ -40,6 +42,11 @@ const openPropFiles = {
   'masks.corner-cuts': MasksCornerCuts,
 };
 
+const hdColorFiles = {
+ 'oklch-colors': OklchColors,
+ 'oklch-hues': OklchHues,
```

Comment:
there is no `oklch-hues` file, so i believe this line is not necessary.

in fact, this whole object could be removed and the logic could be inlined inside the `generateOklchScss` function